pyproject.toml

```diff
@@ -11,3 +12,65 @@ requires = [
     "numpy==1.26.0; python_version=='3.12' and platform_python_implementation != 'PyPy'",
     "numpy<2.0; python_version>='3.13'",
 ]
+build-backend = "setuptools.build_meta"
+
+
+[project]
+name = "pytng"
+description = "Minimal Cython wrapper of the TNG trajectory library"
+readme = "README.rst"
+authors = [{name = "Max Linke, Richard J Gowers, Hugo MacDermott-Opeskin", email = "max_linke@gmx.de"}]
+maintainers = [{name = "MDAnalysis Core Developers", email = "mdanalysis@numfocus.org"}]
```

Comment:
I'm assuming this is the case now?

Reply:
The TNG format is probably dead now, we could probably deprecate this package altogether? But yes

Reply:
I'd be keen to maintain fewer things that are dependencies (even if optional) on the core library - that being said I'm not meant to be the lead maintainer on this, so I can't make those decisions.

In the short term, getting this merged & a release out would be great, it's one of the optional deps that blocks the core library going to 3.12 fully.